.circleci/config.yml

```diff
@@ -1,26 +1,28 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@3.2
+  hmpps: ministryofjustice/hmpps@3.4.0
   slack: circleci/slack@4.4.2
 
+parameters:
+  alerts-slack-channel:
+    type: string
+    default: hmpps_typescript_notifications
+
 executors:
+  builder:
+    docker:
+      - image: 'cimg/node:14.17-browsers'
+    working_directory: ~/app
   integration-tests:
     docker:
-      - image: cimg/node:14.15-browsers
+      - image: cimg/node:14.17-browsers
```

Comment:
noticed we've skipped over `-16` here.  Is that an argument for saying that it should actually be `14-browser` and let it float?

Reply:
as discussed - there isn't a 14-browser as circle recommend pinning to a specific version